### PR TITLE
[CFP-77] FormBuilder migration - Graduated fees

### PR DESCRIPTION
**Pull request by NenehB**
#### What
Continue the migration away from the deprecated design system to the current GOVUK Frontend
#### Ticket

[CFP-66-FormBuilder migration - Graduated fees](https://dsdmoj.atlassian.net/jira/software/c/projects/CFP/boards/624?modal=detail&selectedIssue=CFP-66)

#### Why
Reduce/remove redundant form builders throughout the codebase

--------

#### TODO (wip)

 - [x] Update Validator
 - [x] `:actual_trial_length` validators
 - [x] Update translation 
 - [x] Update spec
 - [x] Update aob

### CODE REVIEW

File: spec/support/shared_examples_for_fee_validators.rb [hunk 1]
```diff
@@ -25,6 +30,33 @@
...
+      expect(fee.errors[:date]).to include(match(/(.*?)(Graduated) fee date cannot be too far in the future/))
+    end
+
+    it 'adds error if before the first repo order date' do
+      allow(claim).to receive(:earliest_representation_order_date).and_return(Date.today)
+      allow(fee).to receive(:claim).and_return(claim)
+
+      fee.date = Date.today - 3.days
+      expect(fee).not_to be_valid
+      expect(fee.errors[:date]).to include(match(/.* date must be no earlier than two years before .*/))
```
Comment (imtiazAhmd) [inline spec/support/shared_examples_for_fee_validators.rb]: As part of the re-writing the error messages, 'must be' needs replacing with  'cannot be'

Comment (imtiazAhmd) [inline spec/support/shared_examples_for_fee_validators.rb]: As part of the re-writing the error messages, 'must be' needs replacing with  'cannot be'

File: config/locales/en/models/claim.yml [hunk 2]
```diff
@@ -132,3 +132,16 @@ en:
...
+          attributes:
+            amount:
+              item_max_amount: The amount for the graduated fee exceeds the limit
+              numericality: Enter a valid amount for the graduated fee
+            date:
+              blank: Enter the graduated fee date
+              check_not_in_future: Graduated fee date cannot be too far in the future
+              check_not_too_far_in_past: Graduated fee date cannot be too far in the past
+              invalid_date: Enter a valid graduated fee date
+              too_long_before_earliest_reporder: The fixed fee date must be no earlier than two years before the earliest representation order date
```
Comment (imtiazAhmd) [inline config/locales/en/models/claim.yml]: `The fixed fee` shouldn't this be `The graduated fee`

Comment (imtiazAhmd) [inline config/locales/en/models/claim.yml:145]: `The fixed fee` shouldn't this be `The graduated fee`

**Review by mpw5 — APPROVED** [review]
LGTM 🎉 

Probably worth getting another review from Imtiaz for the front-end specific changes, but nothing obvious stand out to me.

**Review by imtiazAhmd — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**